IllinoisSocialMediaMacroscope/smm-smile
# 99 allow upload social media data to clowder

To Test:

- Use docker-compose
- Use this PR image 
- set CLOWDER_ON to be true
- Go to history page and try to upload social media data to clowder. You should be able to see files: e.g.
<img width="1645" alt="image" src="https://github.com/IllinoisSocialMediaMacroscope/smm-smile/assets/13950475/7cc9d649-2600-4610-a0f5-4e758961853d">
